tests/PeclJsonSchemaTestSuiteTest.php

```
@@ -0,0 +1,236 @@
+
+    private static function getCheckModeForDraft(string $draft): int
+    {
+        // Match jsonrainbow's check mode for drafts
+        if ($draft === 'draft6') {
+            return Constraint::CHECK_MODE_NORMAL | Constraint::CHECK_MODE_TYPE_CAST;
+        }
+        return Constraint::CHECK_MODE_NORMAL;
+    }
+
+    private static function shouldSkipTest(
+        string $draft,
```

PHPMD (2.15.0)

Avoid unused parameters such as '$draft'. (undefined)

(UnusedFormalParameter)